modules/database/migrations/sqls/20190507194131-add-payment-profiles-up.sql

```diff
@@ -0,0 +1,16 @@
+
+/* Simple payment profile with min collateral and collateral target */
+create table payment_profiles (
+	id bigserial primary key,
+  minimum_maintained_collateral_wei wei_amount default '0',
+  minimum_maintained_collateral_token wei_amount default '0',
+  amount_to_collateralize_wei wei_amount default '0',
+  amount_to_collateralize_token wei_amount default '0'
+);
+
+/* Add payment profile id to the channels column */
+alter table _cm_channels add column payment_profile_id bigint null;
+alter table _cm_channels add constraint payment_profile_id_fk foreign key (payment_profile_id) references payment_profiles(id);
+
+/* Update the channels view */
+create or replace view cm_channels as (select * from _cm_channels);
```

Comment:
We probably honestly shouldn't be doing this, we should only put what we need/use in the view but let's worry about that later.